content/blog/blog_posts.md

```diff
@@ -2,7 +2,7 @@
 title = "Blog posts about Kubeflow"
 description = "Links to interesting blog posts related to Kubeflow"
 weight = 20
-publishDate = 2018-08-12T11:52:19-07:00
+publishDate = 2018-10-27T11:52:19-07:00
```

Comment:
@jlewi by switching this date to 10/27, this shows up on the top as the latest in the https://deploy-preview-274--competent-brattain-de2d6d.netlify.com/blog/.   Is that what you intended? I would assume we want the 0.3 announcement to be the latest blog post. This page lists blog posts from other source talking about Kubeflow.

Reply:
Thanks. I bumped the date to 10-29. It looks like if the date is in the future it doesn't show up. So I just picked a date in the past so it would show up in the preview.